morpholopy/morphology.py

```diff
@@ -358,7 +354,9 @@ def get_axis_lengths_tensor(
         galaxy_log.debug("Total weight of 0, so not calculating axis lengths.")
         return unyt.unyt_array(np.zeros(3), position.units), np.zeros(3)
 
-    Itensor = (weight[:, None, None] / weight.sum()) * np.ones((weight.shape[0], 3, 3))
+    Itensor = (
+        (weight[:, None, None] / weight.sum()) * np.ones((weight.shape[0], 3, 3))
+    ).value
     # Note: unyt currently ignores the position units in the *=
```

Comment:
Does this comment hold true for both the old and new unyt versions?

Reply:
Not anymore, thanks! I removed this comment in the last commit.